extended-tests.json

```diff
@@ -92,44 +92,15 @@
             [ "{?german*}", [ "?11=elf&12=zw%C3%B6lf", "?12=zw%C3%B6lf&11=elf"] ]
         ]
     },
-    "Additional Examples 5: Explode Combinations":{
-        "variables" : {
-            "id" : "admin",
-            "token" : "12345",
-            "tab" : "overview",
-            "keys" : {
-                "key1": "val1",
-                "key2": "val2"
-            }
-        },
-        "testcases":[
-            [ "{?id,token,keys*}", [
-                "?id=admin&token=12345&key1=val1&key2=val2",
-                "?id=admin&token=12345&key2=val2&key1=val1"]
-            ],
-            [ "{/id}{?token,keys*}", [
-                "/admin?token=12345&key1=val1&key2=val2",
-                "/admin?token=12345&key2=val2&key1=val1"]
-            ],
-            [ "{?id,token}{&keys*}", [
-                "?id=admin&token=12345&key1=val1&key2=val2",
-                "?id=admin&token=12345&key2=val2&key1=val1"]
-            ],
-            [ "/user{/id}{?token,tab}{&keys*}", [
-                "/user/admin?token=12345&tab=overview&key1=val1&key2=val2",
-                "/user/admin?token=12345&tab=overview&key2=val2&key1=val1"]
-            ]
-        ]
-    },
```

Comment:
Why are these being removed rather than updated?

Reply:
What are they testing now that we've clarified this?